python/Ganga/Utility/execute.py

```diff
@@ -191,6 +199,10 @@ def execute(command,
         update_env (bool): Should we update the env being passed to what the env was after the command finished running
         return_code (int): This is the returned code from the command which is executed
     """
+    if cwd is None:
+        cwd_ = tempfile.mkdtemp()
+    else:
+        cwd_ = cwd
```

Comment:
Is there any need to create a temporary directory for the `cwd`? Doesn't `subprocess.Popen` just use the current working directory by default?